lib/routes/teams-routes.js

```diff
@@ -0,0 +1,82 @@
+'use strict';
+
+const express = require('express');
+let db = require('../lib/db.js');
+const logger = require('../../logger.js');
+
+let router = express.Router();
+
+/**
+ * Route to request a specific person
+ * @group Teams 
+ * @route GET '/teams'
+ * @security basicAuth
+ * @returns {object} 200 { count: 2, results: [ {}, {} ] }
+ * @returns {Error}  500 - Server error
+ */
+router.get('/', (req, res, next) => {
+  let count = db.teams.length;
+  let results = db.teams;
+  res.json({ count, results });
```

Comment:
This seems to be using a static json-object database from lab 07, instead of our mongo DB models.